### PR TITLE
Snaps E2E Test Upgrades for the new test-snaps UI

### PR DESCRIPTION
**Pull request by bowensanders**
Changes need to be made in order to make existing e2e tests work with the new test-snaps UI (see https://github.com/MetaMask/test-snaps/pull/40) 
This fixes both navigating through the pages as well as changes necessary to the refactor of class & id tags.

### CODE REVIEW

**Comment by bowensanders** [comment]
These test are now working with, and can be co-deployed with https://github.com/MetaMask/test-snaps/pull/40

File: test/e2e/snaps/test-snap-bip-44.spec.js [hunk 1]
```diff
@@ -31,11 +31,16 @@ describe('Test Snap bip-44', function () {
...
-        await driver.fill('.snapId3', 'npm:@metamask/test-snap-bip44');
-        await driver.clickElement({
-          text: 'Connect BIP-44 Snap',
-          tag: 'button',
-        });
+        await driver.delay(1000);
+        await driver.fill('#snapId3', 'npm:@metamask/test-snap-bip44');
+
+        // find and scroll to snapId3
+        const snapButton = await driver.findElement('#snapId4');
```
Comment (FrederikBolding) [inline test/e2e/snaps/test-snap-bip-44.spec.js:38]: This scrolls to `snapId4`, but we are currently working on `snapId3`, is this intentional? If so, we may need a better clarifying comment.

File: test/e2e/snaps/test-snap-managestate.spec.js [hunk 2]
```diff
@@ -70,67 +75,46 @@ describe('Test Snap manageState', function () {
...
         const manageStateResult = await driver.findElement(
-          '.sendManageStateResult',
+          '#sendManageStateResult',
         );
         assert.equal(await manageStateResult.getText(), 'true');
 
         // click get results
-        await driver.waitUntilXWindowHandles(1, 5000, 10000);
-        windowHandles = await driver.getAllWindowHandles();
-        await driver.switchToWindowWithTitle('Test Snaps', windowHandles);
```
Comment (FrederikBolding) [inline test/e2e/snaps/test-snap-managestate.spec.js:89]: How come this isn't needed anymore?

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: test/e2e/webdriver/driver.js [hunk 3]
```diff
@@ -58,6 +58,8 @@ class Driver {
       SPACE: '\uE00D',
       CONTROL: '\uE009',
       COMMAND: '\uE03D',
+      PAGE_DOWN: '\uE00F',
```
[the remark this answers is not in the document]
Comment (bowensanders) [reply test/e2e/webdriver/driver.js:61]: testing artifact. Removing.

[the remark this answers is not in the document]
Comment (bowensanders) [reply test/e2e/webdriver/driver.js:61]: END was also an artifact, before you ask

**Review by FrederikBolding — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**